scripts/predict.py

```diff
@@ -1,46 +1,62 @@
 import json
 import os
+import re
 import pickle
 import sys
 from os.path import dirname, abspath
 sys.path.append(dirname(dirname(abspath(__file__))))
 from pkspace.utils.loaders import PKSpaceLoader # noqa
+from configapp.default_settings import IMAGE_SUFFIX # noqa
 import click # noqa
 
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
-              help='Path where pretrained model')
+              help='Path where pretrained model is located')
+@click.option('--predict_dir', default='.',
+              help='Directory for storing predictions')
 @click.option('--output', default=None,
-              help='Name of output file for predicted output')
-def load_and_predict(mask_path, picture_path, model_path, output):
-    for file in (mask_path, picture_path, model_path):
+              help='Name of file for predicted output')
+def load_and_predict(mask_path, picture_path, model_path, predict_dir, output):
+    for file in (mask_path, picture_path, model_path, predict_dir):
         if not os.path.exists(file):
-            print(file, "Path does not exist")
+            print(file, 'Path does not exist')
             return
+
     if output is None:
-        output = "{}_out.json".format(os.path.splitext(picture_path)[-2])
+        basename = os.path.basename(picture_path)
+        basename = re.sub(IMAGE_SUFFIX + '$', '.json', basename)
```

Comment:
@Adman same as above. Using default `IMAGE_SUFFIX` here does not make much sense, if that value is not easily configurable.